recipe/build.sh

```diff
@@ -25,9 +25,9 @@ autoreconf -vfi
 ./configure --disable-popcnt --disable-clz --enable-generic --enable-tls
 make
 
-programs="addedgeg amtog biplabg catg complg converseg copyg cubhamg deledgeg delptg directg dreadnaut dretodot dretog \
-  genbg genbgL geng genquarticg genrang genspecialg gentourng gentreeg hamheuristic labelg linegraphg listg multig newedgeg \
-  planarg ranlabg showg subdivideg twohamg vcolg watercluster2 NRswitchg"
+programs="addedgeg amtog biplabg catg complg converseg copyg countneg cubhamg deledgeg delptg directg dreadnaut dretodot dretog \
+  genbg genbgL geng genktreeg genquarticg genrang genspecialg gentourng gentreeg hamheuristic labelg linegraphg listg multig newedgeg \
```

Comment:
More programs are missing that were added in 2.8.6 or earlier. Compare with the list in https://github.com/sagemath/sage/pull/36924#issuecomment-1866456706

Reply:
Should be ok with last commit (same list than in Sagemath).